lib/kb_uploadmethods/Utils/ImportFBAModelUtil.py

```diff
@@ -0,0 +1,114 @@
+
+import time
+import json
+import sys
+import uuid
+
+from fba_tools.fba_toolsClient import fba_tools
+from DataFileUtil.DataFileUtilClient import DataFileUtil
+from KBaseReport.KBaseReportClient import KBaseReport
+
+
+def log(message, prefix_newline=False):
+    """Logging function, provides a hook to suppress or redirect log messages."""
+    print(('\n' if prefix_newline else '') + '{0:.2f}'.format(time.time()) + ': ' + str(message))
+
+
+class ImportFBAModelUtil:
+    def __init__(self, config):
+        self.callback_url = config['SDK_CALLBACK_URL']
+        self.token = config['KB_AUTH_TOKEN']
+        self.dfu = DataFileUtil(self.callback_url)
+        self.fba = fba_tools(self.callback_url)
+
+    def import_fbamodel_from_staging(self, params):
+        """
+
+        """
+        log('--->\nrunning {}.{}\n params:\n{}'
+            .format(self.__class__.__name__, sys._getframe().f_code.co_name,
+                    json.dumps(params, indent=1)))
+
+        self._check_param(params, ['model_file', 'file_type', 'workspace_name',
+                                   'model_name'],
+                          ['biomass', 'genome', 'compounds_file'])
+        if not params['file_type'] == 'tsv' and \
```

Comment:
do you mean 'if params['file_type'] == 'tsv' and not params.get('compound_file')'?